cli-commands/flattener/flatten.js

```diff
@@ -31,7 +45,7 @@ const resolveSources = async (file) => {
 	let resolvedFiles
 	try {
 		resolvedFiles = await Profiler.resolveAllSources(resolver, [file], solc)
-	} catch(e) {
+	} catch (e) {
```

Comment:
Can you please remove this catch, as it seems to be unused to me catch(e)-> throw e